server/server.ts

```diff
@@ -6,11 +6,12 @@ import { getEnv } from '@server/utils/environment';
 import { getLogger, configure, isConfigured } from 'log4js';
 import type { Express, Request, Response } from 'express';
 import type { Server } from 'http';
+import cors from 'cors';
 
-import genAI from '@server/routes/genAiRouter';
-import linktaFlow from '@server/routes/linktaFlowRouter';
 import log4jsConfig from '@server/utils/log4js.config.json';
 import { globalErrorHandler } from '@server/middleware/errorHandling';
+import userInput from './routes/userInput';
```

Comment:
@AaronDreyfuss I'm not assuming these were your original variable labels, but let's all take opportunities to improve our variable labels. Here I would propose something like `userInputRouter `and `linktaFlowRouter `for both the exported methods and the file names.